examples/community_tutorials/02_docplex_example.ipynb

```diff
@@ -427,7 +427,7 @@
     "ising_encoding = qubo.ising_model  \n",
     "\n",
     "# Print in a df the ising encoding (we need to remove keys: 'problem_instance' and 'metadata')) \n",
-    "ising_encoding_dict = ising_encoding.asdict(exclude_keys=['problem_instance', 'metadata'])\n",
+    "ising_encoding_dict = ising_encoding.asdict(keys_not_to_include=['problem_instance', 'metadata'])\n",
```

Comment:
I think this addition is wrong: maybe pull changes from dev? @vishal-ph can confirm

Reply:
I had an error and saw this as the new attribute. I pulled from dev before this. Ill try again.